Replay/Replay/ViewController.swift

```diff
@@ -29,12 +27,17 @@ enum PlaybackState {
 class ViewController: UIViewController {
     
     // URL of the Video
-    let media = "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/DesigningForGoogleCast.mp4"
-    var player: AVPlayer? = nil
+    //let media1 = "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/DesigningForGoogleCast.mp4"
+    let media1 = "https://media.w3.org/2010/05/sintel/trailer.mp4"
```

Comment:
media1, media2 and media3 variables are not used. We can construct the array of media directly and remove those 3 media variables.